.github/workflows/README.md

```diff
@@ -1,75 +1,61 @@
-    - An empty GCP project should be used for extra security
-  - to set up dependencies, run the following commands:
-    ```
-    # install kubectl
-    sudo apt-get install -yqq kubectl git
-
-    # install go
-    curl -O https://storage.googleapis.com/golang/go1.12.9.linux-amd64.tar.gz
-    tar -xvf go1.12.9.linux-amd64.tar.gz
-    sudo chown -R root:root ./go
-    sudo mv go /usr/local
-    echo 'export GOPATH=$HOME/go' >> ~/.profile
-    echo 'export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin' >> ~/.profile
-    source ~/.profile
-
-    # install addlicense
-    go get -u github.com/google/addlicense
-    sudo ln -s $HOME/go/bin/addlicense /bin
-
-    # install kind
-    curl -Lo ./kind "https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-$(uname)-amd64" && \
-    chmod +x ./kind && \
-    sudo mv ./kind /usr/local/bin
-
-    # install skaffold
-    curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && \
-    chmod +x skaffold && \
-    sudo mv skaffold /usr/local/bin
-
-    # install docker
-    sudo apt install -yqq apt-transport-https ca-certificates curl gnupg2 software-properties-common && \
-    curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
-    sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" && \
-    sudo apt-get update && \
-    sudo apt-get install -yqq docker-ce && \
-    sudo usermod -aG docker ${USER}
-
-    # logout and back on
-    exit
-    ```
-  - ensure GitHub Actions runs as background service:
-    ```
-    sudo ∼/actions-runner/svc.sh install
-    sudo ∼/actions-runner/svc.sh start
-    ```
-
-
----
-## Workflows
-
-### ci.yaml
-
-#### Triggers
-- commits pushed to master
-- PRs to master
-- PRs to release/ branches
-
-#### Actions
-- ensures kind cluster is running
-- builds all containers in src/
-- deploys local containers to kind
-  - ensures all pods reach ready state
-  - ensures HTTP request to frontend returns HTTP status 200
-- deploys manifests from /releases
-  - ensures all pods reach ready state
-  - ensures HTTP request to frontend returns HTTP status 200
+This page describes the CI/CD workflows for the Online Boutique app, which run in [Github Actions](https://github.com/GoogleCloudPlatform/microservices-demo/actions). 
+
+## Infrastructure 
+
+The CI/CD pipelines for Online Boutique run in Github Actions, using a pool of two [self-hosted runners]((https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-self-hosted-runners)). These runners are GCE instances (virtual machines) that, for every open Pull Request in the repo, run the code test pipeline, deploy test pipeline, and (on master) deploy the latest version of the app to [onlineboutique.dev](https://onlineboutique.dev)
```

Comment:
So do you foresee keeping these self-hosted runners operating for as long as this repo exists?

Do you know if the runners are persistent on GCE VMs?  (When GCE decides to migrate the runner VM, will the self-hosted runner process be uninterrupted and continue to serve requests?)